CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to
 
 ## [Unreleased]
 
+- `GET /statements?agent=` agent no longer accepts `string` type as value. 
+An xAPI Agent with a valid IFI must be passed, formated as a JSON.
```

Comment:
We usually start CHANGELOG entries with a verb. I would suggest the following:

```suggestion
- Force a valid (JSON-formatted) IFI to be passed for the `/statements` GET query `agent` filtering
```

Reply:
This will be done!